src/main/resources/edu/cornell/kfs/module/ar/cu-spring-ar.xml

```diff
@@ -90,5 +90,18 @@
           p:businessObjectService-ref="businessObjectService"
           p:accountingPeriodService-ref="accountingPeriodService"
           p:dateTimeService-ref="dateTimeService" />
+          
+    <bean id="contractsGrantsBillingAwardVerificationService"
+          parent="contractsGrantsBillingAwardVerificationService-parentBean"/>
+    <bean id="contractsGrantsBillingAwardVerificationService-parentBean" abstract="true"
```

Comment:
Was there a particular reason why you had to override the service bean's "parentBean" variant? Could this be simplified to override just the service bean?

Reply:
yes, the simplified spring works, thanks for noticing